propulsion-reactor/Program.fs

```diff
@@ -333,12 +338,24 @@ module Args =
                 | Port _ ->                 "specify a custom port. Uses value of environment variable EQUINOX_ES_PORT if specified. Defaults for Cluster and Direct TCP/IP mode are 30778 and 1113 respectively."
                 | Username _ ->             "specify a username. (optional if environment variable EQUINOX_ES_USERNAME specified)"
                 | Password _ ->             "specify a Password. (optional if environment variable EQUINOX_ES_PASSWORD specified)"
+                | ProjTcp ->                "Request connecting direct to a TCP/IP endpoint for Projection EventStore. Default: Use Clustered mode with Gossip-driven discovery (unless environment variable EQUINOX_ES_PROJ_TCP specifies 'true')."
+                | ProjHost _ ->             "TCP mode: specify Projection EventStore hostname to connect to directly. Clustered mode: use Gossip protocol against all A records returned from DNS query. (optional if environment variable EQUINOX_ES_PROJ_HOST specified)"
+                | ProjPort _ ->             "specify Projection EventStore custom port. Uses value of environment variable EQUINOX_ES_PROJ_PORT if specified. Defaults for Cluster and Direct TCP/IP mode are 30778 and 1113 respectively."
+                | ProjUsername _ ->         "specify a username for Projection EventStore. (optional if environment variable EQUINOX_ES_PROJ_USERNAME specified)"
```

Comment:
should/could this fall back to the value of `EQUINOX_ES_USERNAME` so the common case where they are the same cluster are easy to wire up? (and same for others)

Reply:
Agreed

Reply:
Check for Argument `-u` -> check for env. variable `EQUINOX_ES_PROJ_USERNAME` -> fallback to __.user